src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -588,14 +589,17 @@ def _prepare_attn_mask(self, attention_mask, input_shape, inputs_embeds, past_ke
         # create causal mask
         # [bsz, seq_len] -> [bsz, 1, tgt_seq_len, src_seq_len]
         combined_attention_mask = None
+        device = attention_mask.device
+        dtype = torch.bool
```

Comment:
If this is hard-coded, it shouldn't be an argument in the function where it's send.